SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/Models/Log.cs

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please note that files with "Ingestoin/Models" in the path are auto generated by `autorest`.